src/templates/docPage.js

```diff
@@ -130,6 +130,7 @@ export const pageQuery = graphql`
       body
       frontmatter {
         title
+        shortDescription
```

Comment:
Should this get passed into the `description` field on the `SEO` component on [line 67](https://github.com/newrelic/docs-website/pull/895/files#diff-1d1e91cf1df9133860992dc5d7fb0566a86cbae77ff1c92aea195586dc730f4dL67)?

Reply:
Yep